test.js

```diff
@@ -141,6 +141,17 @@ test('removeQueryParameters boolean `false` option', t => {
 	t.is(normalizeUrl('www.sindresorhus.com?foo=bar&utm_medium=test&ref=test_ref', options), 'http://www.sindresorhus.com/?foo=bar&ref=test_ref&utm_medium=test');
 });
 
+test('keepQueryParameters option', t => {
+	const options = {
+		stripWWW: false,
+		removeQueryParameters: false,
```

Comment:
These are unrelated to what's being tested.

Reply:
Yes, `stripWWW: false` is unrelated. I'll remove it.

If we don't set `removeQueryParameters` to `false`, the default value `([/^utm_\w+/i])` tries to remove `utm_medium` while `keepQueryParameters` tries to keep it. This conflict makes the test fail.